vignettes/Model.Rmd

```diff
@@ -266,28 +312,42 @@ The remaining vignettes describe how to adjust sets of parameters through a numb
 
     -   `peak_season_offset()`: correlating timed interventions with seasonal malaria
 
-5.  [Vaccines](https://mrc-ide.github.io/malariasimulation/articles/Vaccines.html)
+4.  [Vaccines](https://mrc-ide.github.io/malariasimulation/articles/Vaccines.html)
 
     -   `set_mass_pev()`: implementation of a pre-erythrocytic vaccination intervention
-    -   `set_pev_epi()`: Implementation of a pre-erythrocytic vaccination intervention from a certain age
+    
+    -   `set_pev_epi()`: implementation of a pre-erythrocytic vaccination intervention from a certain age
+    
     -   `set_tbv()`: implementation of a transmission blocking vaccination intervention
 
-6.  [Vector Control: Bednets](https://mrc-ide.github.io/malariasimulation/articles/VectorControl.html)
+5.  [Vector Control: Bednets](https://mrc-ide.github.io/malariasimulation/articles/VectorControl_Bednets.html)
 
     -   `set_bednets()`: implementation of bednet distribution intervention
 
-7.  [Vector Control: Indoor Residual Spraying](https://mrc-ide.github.io/malariasimulation/articles/VectorControl.html)
+6.  [Vector Control: Indoor Residual Spraying](https://mrc-ide.github.io/malariasimulation/articles/VectorControl_IRS.html)
 
     -   `set_spraying()`: implementation of an indoor residual spraying intervention
 
-8.  [Matching PfPR2-10 to EIR](https://mrc-ide.github.io/malariasimulation/articles/EIRprevmatch.html)
+7.  [Mosquito Species](https://mrc-ide.github.io/malariasimulation/articles/SetSpecies.html)
+
+    -   `set_species()`: setting mosquito distribution
+
+8.  [Carrying Capacity](https://mrc-ide.github.io/malariasimulation/articles/Carrying-capacity.html)
+
+    -   `set_carrying_capacity()`: changes mosquito carrying capacity, e.g. to model larval source management impact
+    
+9.  [Matching PfPR2-10 to EIR](https://mrc-ide.github.io/malariasimulation/articles/EIRprevmatch.html)
 
     -   Using *Pf*PR~2-10~ data to estimate EIR
 
 9.  [Metapopulation Modelling](https://mrc-ide.github.io/malariasimulation/articles/Metapopulation.html)
 
-    -   `run_metapop_simulation()`: to model multiple areas simultaneously
+    -   `run_metapop_simulation()`: model multiple areas simultaneously
 
-10. [Variation](https://mrc-ide.github.io/malariasimulation/articles/Variation.html)
+11. [Stochastic Variation](https://mrc-ide.github.io/malariasimulation/articles/Variation.html)
 
     -   `run_simulation_with_repetitions()`: running simulations with replicates
+    
+12. [Parameter Variation](https://mrc-ide.github.io/malariasimulation/articles/ParameterVariation.html)
```

Comment:
This looks great, would just update the parameter variation link so it clicks through: https://mrc-ide.github.io/malariasimulation/articles/Parameter_variation.html

Reply:
I might check all of the links in this portion of the vignette again-- for me, the links take me to version 1.4.1 (erroring out if it is a new vignette not created in that version). May just be the way the site is rendering for me, if so disregard!

Reply:
It won't click through at the moment because the real link doesn't exist. It should have internal consistency and work when it gets published, though.

Reply:
Ah got it, that makes sense!